Calculator/ViewController.swift

```diff
@@ -0,0 +1,197 @@
+//
+//  ViewController.swift
+//  Calculator
+//
+//  Created by Nguyen Cuong on 7/3/19.
+//  Copyright © 2019 Nguyen Cuong. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    @IBOutlet var buttons: [UIButton]!
+    @IBOutlet weak var buttonAC: UIButton!
+    @IBOutlet weak var showCalculator: UILabel!
+    var calculation:Bool = false
```

Comment:
```suggestion
    var calculation = false
```
Tìm hiểu type inference